src/trackExtensions.ts

```diff
@@ -1,5 +1,7 @@
 import { Extension, getExtensionHash, getLogs } from '@colony/colony-js';
 
+import { addVotingReputationParamsToDB } from '~handlers/extensions';
```

Comment:
Nitpick, but having to import it from `~handlers` suggests to me it could live in `~utils` instead